examples/getstarted/.gitignore

```diff
@@ -101,6 +101,8 @@ node_modules
 
 testApp
 coverage
+!src/api/coverage
```

Comment:
Instead of adding exception everywhere it would be useful we can just ignore a root /coverage folder instead that would be easier to maintain I think :)

Reply:
That would be much better. Definitely agreed @alexandrebodin; will circle back on this one if someone else does not beat me to it

Reply:
I think 90db0788f1c60744df06ff432fe7d65acae81587 solves it?